src/common/com/intellij/plugins/haxe/model/HaxeAbstractClassModel.java

```diff
@@ -53,4 +56,17 @@ public HaxeAbstractClassDeclaration getAbstractClass() {
   protected HaxeAbstractBody getAbstractClassBody() {
     return getAbstractClass().getAbstractBody();
   }
+
+  public boolean isForwarded(String name) {
+    final HaxeMacroClass forwardMeta = getMeta(FORWARD);
+    if (forwardMeta != null) {
+      HaxeCustomMeta customMeta = forwardMeta.getCustomMeta();
+      final HaxeExpressionList expressionList = customMeta.getExpressionList();
+      if (expressionList == null) return true;
+      for (HaxeExpression expression : expressionList.getExpressionList()) {
+        if (expression.getText().equals(name)) return true;
```

Comment:
How does this work with `@:forward()` (empty list)?  Will returning false in that case be OK?  Actually, I think, yes.  But...

`@:forward` without arguments forwards all fields.  `@:forward()` without any arguments won't forward any fields.  I think this is not what the user would expect, so we should flag that as an error -- or, at least, a warning.

Reply:
Ah, yeah, that's a mistake